kie-server-parent/kie-server-remote/kie-server-client/src/build/revapi-config.json

```diff
@@ -1,57 +1,67 @@
 {
-    "filters": {
-        "revapi": {
-            "java": {
-                "_comment": "We don't want to check classes, just interfaces. Every class is therefore excluded so new interfaces and classes in future are discovered by revapi.",
-                "filter": {
-                    "packages": {
-                        "_comment": [
-                            "We don't want to check transitive classes, e.g. from kie-server-api, since we already check them in their own module.",
-                            "Therefore, only module's packages are included. Excluded packages contain only classes."
-                        ],
-                        "regex": true,
-                        "include": [
-                            "org\\.kie\\.server\\.client.*"
-                        ],
-                        "exclude": [
-                            "org\\.kie\\.server\\.client\\.admin\\.impl",
-                            "org\\.kie\\.server\\.client\\.balancer\\.impl",
-                            "org\\.kie\\.server\\.client\\.credentials",
-                            "org\\.kie\\.server\\.client\\.impl"
-                        ]
-                    },
-                    "classes": {
-                        "_comment": "In the remaining packages which are mixed (both interfaces and classes), we just exclude classes.",
-                        "regex": false,
-                        "exclude": [
-                            "org.kie.server.client.balancer.LoadBalancer",
-                            "org.kie.server.client.helper.CaseServicesClientBuilder",
-                            "org.kie.server.client.helper.DroolsServicesClientBuilder",
-                            "org.kie.server.client.helper.JBPMServicesClientBuilder",
-                            "org.kie.server.client.helper.JBPMUIServicesClientBuilder",
-                            "org.kie.server.client.helper.OptaplannerServicesClientBuilder",
-                            "org.kie.server.client.KieServicesException",
-                            "org.kie.server.client.KieServicesFactory"
-                        ]
-                    }
-                }
-            }
+  "filters": {
```

Comment:
Can we keep original formatting of the json file?

Reply:
+1